src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -0,0 +1,425 @@
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@material/mwc-button/mwc-button";
+import "@material/mwc-icon-button/mwc-icon-button";
+import { mdiRefresh } from "@mdi/js";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { debounce } from "../../../../../common/util/debounce";
+import "../../../../../components/ha-card";
+import "../../../../../components/ha-svg-icon";
+import "../../../../../components/ha-icon-next";
+import "../../../../../components/ha-switch";
+import {
+  fetchNodeConfigParameters,
+  setNodeConfigParameter,
+  ZWaveJSNodeConfigParams,
+} from "../../../../../data/zwave_js";
+import "../../../../../layouts/hass-tabs-subpage";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant, Route } from "../../../../../types";
+import "../../../ha-config-section";
+import { configTabs } from "./zwave_js-config-router";
+import {
+  DeviceRegistryEntry,
+  computeDeviceName,
+  subscribeDeviceRegistry,
+} from "../../../../../data/device_registry";
+import { SubscribeMixin } from "../../../../../mixins/subscribe-mixin";
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import memoizeOne from "memoize-one";
+
+const getDevice = memoizeOne(
+  (entries: DeviceRegistryEntry[], deviceId: string) =>
+    entries.find((device) => device.id === deviceId)
+);
+
+const getIdentifier = memoizeOne((device: DeviceRegistryEntry) =>
+  device.identifiers.find((ident) => ident[0] === "zwave_js")
+);
+
+@customElement("zwave_js-node-config")
+class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
+  @property({ type: Object }) public hass!: HomeAssistant;
+
+  @property({ type: Object }) public route!: Route;
```

Comment:
We wouldn't want to support being set as an attribute. Better just set `attribute: false`
```suggestion
  @property({ attribute: false }) public hass!: HomeAssistant;

  @property({ attribute: false }) public route!: Route;
```